dotCMS/src/integration-test/java/com/dotcms/publishing/PublisherFilterImplTest.java

```diff
@@ -0,0 +1,122 @@
+package com.dotcms.publishing;
+
+import com.dotcms.publisher.util.PusheableAsset;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.util.UUIDGenerator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PublisherFilterImplTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
```

Comment:
use this doc guide
```

/**
* Method to test: include this in case the method to test does not belong to the class this test-class is testing 
* Given Scenario: Scenario under test
* ExpectedResult: Expected result that the assertion is checking against
*
*/
```